docs/Sprint-Review/sprint-93-summary.md

```diff
@@ -0,0 +1,80 @@
+# Sprint 93 Summary
+
+02/14/2024 - 02/27/2024
+
+Velocity (Dev): 18
+
+## Sprint Goal
+* Dev:
+    * Continue parsing engine development and begin work on enhancement tickets
+        * #2536 Cat 4 validation 
+    * #1858 Secure OFA staff access to Kibana 
+        * Unblocks #1350 when complete 
+* DevOps:
+    * #2790 - Update deployment code to support Kibana and integrate with Standing Elastic instance
+* Design: 
+    * Tie up current documentation work
+    * Continue refinement of research roadmap
+
+
+## Tickets
+### Completed/Merged
+* [#2781 As a developer,  I want to have documentation on django migration best practices](https://app.zenhub.com/workspaces/sprint-board-5f18ab06dfd91c000f7e682e/issues/gh/raft-tech/tanf-app/2781)
+* [#2813 Reduce dev environment count](https://app.zenhub.com/workspaces/sprint-board-5f18ab06dfd91c000f7e682e/issues/gh/raft-tech/tanf-app/2813)
+* [#2790 Kibana deployment](https://app.zenhub.com/workspaces/sprint-board-5f18ab06dfd91c000f7e682e/issues/gh/raft-tech/tanf-app/2790)
+* [#2824 Resolve issues raised in pen testing findings](https://app.zenhub.com/workspaces/sprint-board-5f18ab06dfd91c000f7e682e/issues/gh/raft-tech/tanf-app/2824)
+* [#2832 Accessibility Guide update](https://github.com/raft-tech/TANF-app/pull/2832)
+* [#2794 Make Case Number validators accept spaces and special characters](https://github.com/raft-tech/TANF-app/issues/2794)
+* [#2771 Update validateSSN to allow for all 0s, 1s,...9s as valid values except in cat3 scenarios](https://github.com/raft-tech/TANF-app/issues/2771)
+* [#2855 Update Failed Deployment Troubleshooting](https://github.com/raft-tech/TANF-app/pull/2855)
+* [#2768 Fix production OWASP scan reporting](https://app.zenhub.com/workspaces/sprint-board-5f18ab06dfd91c000f7e682e/issues/gh/raft-tech/tanf-app/2768)
```

Comment:
may want to track somewhere that #2768 is still open. but the initial fix was indeed merged in sprint 93.

Reply:
Do you think the [roadmap item](https://app.mural.co/t/raft2792/m/raft2792/1675268418098/d56ec0d5415d8cda9c6ae9cae61154bcc4e74bba?wid=2-1705963634552) covers that or that we need another tracker?

Reply:
yeah thats fine to track there.